### PR TITLE
Merge third_party/metrics_proto dependencies

### PR DESCRIPTION
**Pull request by joeltine**

### CODE REVIEW

File: third_party/metrics_proto/METADATA [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    value: "https://chromium.googlesource.com/chromium/src/third_party/metrics_proto/"
+  }
+  version: "fd3a73980b82eea000403c2c9c1f204f6612e7cf"
+  last_upgrade_date {
+    year: 2018
+    month: 8
+    day: 30
+  }
+  license_type: NOTICE
+}
```
Comment (kaidokert) [inline third_party/metrics_proto/METADATA:17]: tiny nit: looks like missing line end. Our pre-submits do not check anything in third_party/ by default, so it wouldn't alert on it.

**Review by kaidokert — APPROVED** [review]
lgtm - would be good to have @briantting take a quick look too

File: third_party/metrics_proto/METADATA (hunk 1 above)
Comment (joeltine) [reply third_party/metrics_proto/METADATA]: Done

File: third_party/metrics_proto/OWNERS [hunk 2]
```diff
@@ -0,0 +1,7 @@
+file://base/metrics/OWNERS
+
+per-file sampled_profile.proto=wittman@chromium.org
```
Comment (joeltine) [inline third_party/metrics_proto/OWNERS:3]: Should I change these too?

File: third_party/metrics_proto/PRESUBMIT.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""Presubmit script for metrics_proto.
+
+See http://dev.chromium.org/developers/how-tos/depottools/presubmit-scripts
+for more details on the presubmit API built into gcl.
+"""
+
+README = 'README.chromium'
```
Comment (joeltine) [inline third_party/metrics_proto/PRESUBMIT.py:11]: Not sure if this file is helpful either...?

File: third_party/metrics_proto/OWNERS (hunk 2 above)
Comment (kaidokert) [reply third_party/metrics_proto/OWNERS:3]: No, let's leave upstream OWNERs in place, we don't actually have an override in Cobalt for those ( ongoing discussion how and what we should be doing there )

File: third_party/metrics_proto/PRESUBMIT.py (hunk 3 above)
Comment (kaidokert) [reply third_party/metrics_proto/PRESUBMIT.py:11]: We aren't using those, as `PRESUBMIT.py` is a `depot_tools` construct, and we migrated away from DT. But there's also no reason to delete them, better to leave as is and modify upstream directory imports as little as possible.

Comment (joeltine) [reply third_party/metrics_proto/PRESUBMIT.py:11]: ack

**Review by briantting — APPROVED** [review]

File: third_party/metrics_proto/METADATA [hunk 4]
```diff
@@ -0,0 +1,17 @@
+name: "metrics_proto"
+description:
+  "These are the proto definitions to upload metrics data to Google."
+
+third_party {
+  url {
+    type: GIT
+    value: "https://chromium.googlesource.com/chromium/src/third_party/metrics_proto/"
+  }
+  version: "fd3a73980b82eea000403c2c9c1f204f6612e7cf"
```
Comment (dahlstrom-g) [inline third_party/metrics_proto/METADATA:10]: draft comment for pending review I'm about to cancel

Comment (dahlstrom-g) [reply third_party/metrics_proto/METADATA:10]: Yeah, this is another total failure of the GitHub PR UI. 😠

**Pull request merged**